liblangutil/SemVerHandler.cpp

```diff
@@ -37,7 +37,7 @@ SemVerVersion::SemVerVersion(string const& _versionString)
 	{
 		unsigned v = 0;
 		for (; i != end && '0' <= *i && *i <= '9'; ++i)
-			v = v * 10 + (*i - '0');
+			v = v * 10 + unsigned(*i - '0');
```

Comment:
Does this work for char both being signed and unsigned?

Reply:
Changed to named cast instead